### DISCUSSION TITLE
[Featurerequest] 90 Grad drehung der Kurse

### DISCUSSION THREAD

**Discussion by GSGderMicha** [Mogli12/KeyboardSteer#758] [Ideas]
Hi Mogli,

vielen Dank für den Mod ich finde ihn absolut geil und freue mich über die Verbesserungen gegenüber dem LS19.

Wie im Titel zu lesen, würde ich mir für deinen Mod noch Wünschen, dass man die Spuren per Knopfdruck um 90 grad drehen kann, für zum Beispiel Vorgewende bahnen etc.

Gruß der Micha

**Comment by Mogli12** [comment]
Hallo Micha,

ja, das wäre wirklich interessant, ist im Detail aber schwierig. Damit man es richtig nutzen kann, müsste VCA sich für alle 4 Richtungen die letzte Spur merken. Und wenn man das bauen muss, dann wäre es auch schön, wenn man die Spuren für alle 4 Richtungen getrennt anpassen könnte.

Aber zuerst möchte ich die aktuelle Version fertig machen.

Viele Grüße,
Stefan

**Comment by GSGderMicha** [comment]
Hallo Stefan, 

ich habe gerade gesehen, dass man ja im Menü die Bahn automatisch um 90 grad drehen kann... klar ist es nicht nur ein Knopfdruck aber im grunde das was ich meinte. Mach weiter so wie du es möchtest, bis jetzt hast du ja schon echt Mega gute Arbeit geleistet!

Grüße
